### PR TITLE
Clean up OverallArchitecture, Fuzzing and Logging docs

### PR DESCRIPTION
**Pull request by olganaumenko**
## Description

- Minor bugs fixed: wording, formatting, punctuation.
- _Logging_ section added to [OverallArchitecture.md](https://github.com/UnitTestBot/UTBotJava/blob/main/docs/OverallArchitecture.md).
- _Logging.md_ moved to [contributing](https://github.com/UnitTestBot/UTBotJava/tree/main/docs/contributing) folder.

## How to test

The PR introduces no code changes.
Manual scenario: the text itself has been previewed in the IntelliJ IDEA viewer.

## Self-check list

- [x] I've set the proper **labels** for my PR (at least, for category and component).
- [x] PR **title** and **description** are clear and intelligible.
- [ ] I've added enough **comments** to my code, particularly in hard-to-understand areas.
- [ ] The functionality I've repaired, changed or added is covered with **automated tests**.
- [x] **Manual tests** have been provided optionally.
- [ ] The **documentation** for the functionality I've been working on is up-to-date.

### CODE REVIEW

File: docs/Fuzzing Platform.md [hunk 1]
```diff
@@ -42,26 +48,34 @@ The general fuzzing process gets the list of parameter descriptions as input and
...
+`(-3, false), (0, false), (10, false), (-3, true), (0, true), (10, true)`.
+Depending on the programming language,
+one may use interface descriptions or annotations (type hints) instead of defining the specific type.
+Fuzzing platform (FP) is not able to create the concrete objects as it does not deal with the specific languages.
+It can still convert the descriptions to the known constructs it can work with.
+
+Say, in most of the programming languages, any integer may be represented as a bit array, and the fuzzer can construct and 
+modify bit arrays. So, in the general case, the boundary values for the integer are these bit arrays:
+
+* [0, 0, 0, ..., 0] — null
```
Comment (Markoutte) [inline docs/Fuzzing Platform.md:62]: Could we change the `null` to `zero`?

File: docs/Fuzzing Platform.md [hunk 2]
```diff
@@ -42,26 +48,34 @@ The general fuzzing process gets the list of parameter descriptions as input and
...
+* [1, 1, 1, ..., 1] — minus 1
 
 One can correctly use this representation for unsigned integers as well:
 
-* [0, 0, 0, ..., 0] - null (minimum value)
-* [1, 0, 0, ..., 0] - maximum value / 2
-* [0, 1, 1, ..., 1] - maximum value / 2 + 1
-* [0, 0, ..., 0, 1] - plus 1
-* [1, 1, 1, ..., 1] - maximum value
+* [0, 0, 0, ..., 0] — null (minimum value)
```
Comment (Markoutte) [inline docs/Fuzzing Platform.md:70]: Could we change the null to `zero`?

**Review by Markoutte — APPROVED** [review]

File: docs/Fuzzing Platform.md (hunk 1 above)
Comment (olganaumenko) [reply docs/Fuzzing Platform.md]: Thank you! Fixed.

**Review by Domonion — APPROVED** [review]
lgtm

**Pull request merged**